### PR TITLE
Add ability to copy a content object

### PR DESCRIPTION
**Pull request by elv-zenia**
Allow user to copy a content object. See associated client-js PR: https://github.com/eluv-io/elv-client-js/pull/128.
The content browser modal is identical to the one used by live-manager.

Main test cases:
- Copying an object with no encryption keys
- Copying an object with kms and user caps keys
- Copying an object with one or the other ^
- File download flow for the owner of a copied object
- File download flow for the non-owner (member of an admin group) of a copied object
- Encrypted video files successfully download and play

image

https://user-images.githubusercontent.com/91760982/141022944-f78c82f0-ac70-4639-bdb7-8e8b7ffb2b39.mov

https://user-images.githubusercontent.com/91760982/141022953-f1bc7718-340a-4943-9561-c5364251ce72.mov

Relates to #8 

Future improvements:
- Update sass-lint (disable-next-line is in latest update)
- Update mobx and mobx-react (for the use of makeAutoObservable and makeObservable). Updating these packages broke other pages so I think this should be a separate change to fix any code that needs to be updated.

### CODE REVIEW

**Comment by elv-zenia** [comment]
@elv-kevin Can you review? I think including fields to edit (like name) needs some extra work. Like maybe a multi-step modal. Also, looks like copying to a different library works as well.

File: src/stores/Content.js [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+  @observable libraries = undefined;
+  contentTypesPromise = undefined;
+  tenantObjectId = undefined;
+  tenant = undefined;
+
+  @computed get client() {
+    return this.rootStore.client;
+  }
+
+  @computed get events() {
```
Comment (elv-kevin) [inline src/stores/Content.js:16]: Can you remove this and anything else that isn't used?

**Review by elv-kevin — APPROVED** [review]
This looks really good, really neat to see it working.

Can you add one thing: a confirmation when you select the library to copy the object into. You can use the existing Confirm component (search the src for usage). Just have it say 'are you sure you want to copy 'object name' into 'library name'?

**Comment by elv-kevin** [comment]
one last nitpick before I merge it - hitting cancel on the confirmation shouldn't close the whole thing, just the modal. I think all you need to do is move Close() into the onConfirm function instead of unconditionally after the confirm.

**Pull request merged**